### PR TITLE
 config: enable tichi for pingcap/tidb-binlog

### LINKED ISSUE #346: REQUEST: deploy prow for tidb-binlog
### GitHub Repo Name
pingcap/tidb-binlog

### Which plugins do you want to deploy
- hold
- wip
- tide
- lifecycle
- cherrypicker

### Requirements
- [x] I have carefully read the instructions for the plugin I want to use.
- [x] I agree to leave the plugin configuration to be maintained by sig-ti-community-infra.

### TODO
- [x] @ti-community-infra/sig-community-infra  Add the requested plugin configuration
- [ ] @ti-community-infra/sig-community-infra  Add repository admin to Robots
- [ ] @ti-community-infra/sig-community-infra  Turn off the old robot function

### PR DESCRIPTION
**Pull request by Mini256**
close #346

### CODE REVIEW

File: prow/config/config.yaml [hunk 1]
```diff
@@ -529,6 +529,48 @@ branch-protection:
...
+            master:
+              protect: true
+              required_status_checks:
+                contexts:
+                  - 'idc-jenkins-ci-binlog/build'
+                  - 'idc-jenkins-ci-binlog/check'
+                  - 'idc-jenkins-ci-tidb-binlog/integration-test'
+                  - 'idc-jenkins-ci-tidb-binlog/check_release_note'
+                  - 'idc-jenkins-ci-tidb-binlog/unit-test'
+                strict: true
```
Comment (hi-rustin) [inline prow/config/config.yaml:543]: If we don't use the tars, I think it should be false.

File: prow/config/config.yaml [hunk 2]
```diff
@@ -529,6 +529,48 @@ branch-protection:
...
+            release-4.0:
+              protect: true
+              required_status_checks:
+                contexts:
+                  - 'idc-jenkins-ci-binlog/build'
+                  - 'idc-jenkins-ci-binlog/check'
+                  - 'idc-jenkins-ci-tidb-binlog/integration-test'
+                  - 'idc-jenkins-ci-tidb-binlog/check_release_note'
+                  - 'idc-jenkins-ci-tidb-binlog/unit-test'
+                strict: true
```
Comment (hi-rustin) [inline prow/config/config.yaml:553]: ditto

File: prow/config/config.yaml [hunk 3]
```diff
@@ -529,6 +529,48 @@ branch-protection:
...
+            release-5.0:
+              protect: true
+              required_status_checks:
+                contexts:
+                  - 'idc-jenkins-ci-binlog/build'
+                  - 'idc-jenkins-ci-binlog/check'
+                  - 'idc-jenkins-ci-tidb-binlog/integration-test'
+                  - 'idc-jenkins-ci-tidb-binlog/check_release_note'
+                  - 'idc-jenkins-ci-tidb-binlog/unit-test'
+                strict: true
```
Comment (hi-rustin) [inline prow/config/config.yaml:563]: ditto

File: prow/config/config.yaml [hunk 4]
```diff
@@ -529,6 +529,48 @@ branch-protection:
...
+            release-5.1:
+              protect: true
+              required_status_checks:
+                contexts:
+                  - 'idc-jenkins-ci-binlog/build'
+                  - 'idc-jenkins-ci-binlog/check'
+                  - 'idc-jenkins-ci-tidb-binlog/integration-test'
+                  - 'idc-jenkins-ci-tidb-binlog/check_release_note'
+                  - 'idc-jenkins-ci-tidb-binlog/unit-test'
+                strict: true
```
Comment (hi-rustin) [inline prow/config/config.yaml:573]: ditto

**Review by hi-rustin — CHANGES_REQUESTED** [review]
It looks like bin-logs used to require merge base, so we need to enable tars?

**Review by lichunzhu — APPROVED** [review]
LGTM

File: prow/config/config.yaml (hunk 1 above)
Comment (Mini256) [reply prow/config/config.yaml:543]: What is the reason for setting to false?

Comment (hi-rustin) [reply prow/config/config.yaml:543]: This is whether to request merge base.

**Review by hi-rustin — APPROVED** [review]

**Pull request merged**